Src/ReadCombustionTankPressure.c

```diff
@@ -0,0 +1,69 @@
+#include "stm32f4xx.h"
+#include "stm32f4xx_hal_conf.h"
+#include "cmsis_os.h"
+#include "math.h"
+
+#include "ReadCombustionTankPressure.h"
+
+#include "Data.h"
+
+static int READ_COMBUSTION_TANK_PRESSURE_PERIOD = 300;
+
+static const int ADC_POLL_TIMEOUT = 150;
+
+void readCombustionTankPressureTask(void const* arg)
+{
+    CombustionTankPressureData* data = (CombustionTankPressureData*) arg;
+    uint32_t prevWakeTime = osKernelSysTick();
+
+    uint16_t adcRead = 0;   // Stores a 12 bit value ( maximum resolution for this ADC )
+    double vo = 0;  // The voltage across the 133k resistor
+    double vi = 0;  // The pressure sensor output
+    double tankPressure = 0;
+
+    int counter = 0;    // Counts up to 5 ADC readings before averaging
+
+    // Resistor values in kOhms
+    int R1 = 100;
+    int R2 = 133;
```

Comment:
not a constant, shoudl be lower cas.e

Reply:
Is it better to make them constants, because they will not change while code is running? I could even put them straight into the equation and not even use variables.